src/page-analysis/background/get-fav-icon.js

```diff
@@ -1,20 +1,26 @@
+import favicon from 'favicon'
```

Comment:
Yes it does do another fetch. I was thinking how to get favicon if you're given a URL. Better to think how to get it if you're given the DOM, as we already have that in both contexts. Something like: `document.querySelector("link[rel*='icon']").href` should work if the page has a favicon.